experanto/interpolators.py

```diff
@@ -327,8 +331,10 @@ def interpolate(self, times: np.ndarray) -> tuple[np.ndarray, np.ndarray]:
         # Go through files, load them and extract all frames
         unique_file_idx = np.unique(data_file_idx)
         out = np.zeros([len(valid_times)] + list(self._image_size))
+        meta = None
```

Comment:
1) I am not sure if `meta` should be in the `interpolate` function in the first place. Why is it here now?
2) if you have several `unique_file_idx` - the `data` would be appended in `out` but for `meta` only the last point will be returned, which is probably incorrect behaviour. Or am I missing sth?